src/modules/sni/item.cpp

```diff
@@ -300,10 +300,6 @@ void Item::updateImage() {
   auto pixbuf = getIconPixbuf();
   auto scaled_icon_size = getScaledIconSize();
 
-  if (!pixbuf) {
-    pixbuf = getIconByName("image-missing", getScaledIconSize());
-  }
-
```

Comment:
I noticed that this part is unnecessary as pixbuf cannot be `false`, as `getIconPixbuf()` will always either 
* return a valid Pixbuf, or 
* throw an exception.